docs/rancher/resource-quota.md

```diff
@@ -60,16 +60,16 @@ For more information, see [Memory Overhead](https://kubevirt.io/user-guide/virtu
+## Automatic adjustment of ResourceQuota during migration
 When the allocated resource quota controlled by the `ResourceQuota` object reaches its limit, migrating a VM becomes unfeasible. The migration process automatically creates a new pod mirroring the resource requirements of the source VM. If these pod creation prerequisites surpass the defined quota, the migration operation cannot proceed.
 
 _Available as of v1.2.0_
 
-In Harvester, the `ResourceQuota` values dynamically expand ahead of migration to accommodate the resource needs of the target virtual machine. After migration, the ResourceQuotas are reinstated to their prior configurations.
+In Harvester, the `ResourceQuota` values will dynamically expand ahead of migration to accommodate the resource needs of the target virtual machine. After migration, the ResourceQuotas will be reinstated to their prior configurations.
 
-Please be aware of the following considerations of the automatic resizing of `ResourceQuota`:
-- When raising the `ResourceQuota` value, if you create, start, or restore other VMs, Harvester will verify if the resources are sufficient based on the original `ResourceQuota`. If the conditions are not met, the system will alert that the migration process is not feasible.
+Please be aware of the following constrains of the automatic resizing of `ResourceQuota`:
 - `ResourceQuota` cannot be changed during VM migration.
-- After expanding `ResourceQuota`, potential resource contention may emerge between non-VM pods and VM pods, leading to migration failures. Consequently, deploying non-VM pods to the same namespace is not recommended.
-- Due to the concurrent limitation of the webhook validator, the VM controller will execute a secondary validation to confirm resource sufficiency. If the resources are insufficient, it will auto config the VM's`RunStrategy` to **Halted**, and a new annotation `harvesterhci.io/insufficient-resource-quota: xxxxx` will be added to the VM object, informing you that the VM was shut down due to insufficient resources.
+- When raising the `ResourceQuota` value, if you create, start, or restore other VMs, Harvester will verify if the resources are sufficient based on the original `ResourceQuota`. If the conditions are not met, the system will alert that the migration process is not feasible.
+- After expanding `ResourceQuota`, potential resource contention may occur between non-VM pods and VM pods, leading to migration failures. Therefore, deploying custom container workloads and VMs to the same namespace is not recommended.
+- Due to the concurrent limitation of the webhook validator, the VM controller will execute a secondary validation to confirm resource sufficiency. If the resource is insufficient, it will auto config the VM's`RunStrategy` to `Halted`, and a new annotation `harvesterhci.io/insufficient-resource-quota` will be added to the VM object, informing you that the VM was shut down due to insufficient resources.
```

Comment:
```suggestion
- Due to the concurrent limitation of the webhook validator, the VM controller will execute a secondary validation to confirm resource sufficiency. If the resource is insufficient, it will auto config the VM's `RunStrategy` to `Halted`, and a new annotation `harvesterhci.io/insufficient-resource-quota` will be added to the VM object, informing you that the VM was shut down due to insufficient resources.
```